controllers/argocd/custommapper_test.go

```diff
@@ -153,7 +153,7 @@ func TestReconcileArgoCD_tlsSecretMapperRepoServer(t *testing.T) {
 		resObjs := []client.Object{argocd, secret, service}
 		subresObjs := []client.Object{argocd}
 		runtimeObjs := []runtime.Object{}
-		sch := makeTestReconcilerScheme(argoproj.AddToScheme, configv1.AddToScheme, routev1.AddToScheme)
+		sch := makeTestReconcilerScheme(argoproj.AddToScheme, configv1.Install, routev1.Install)
```

Comment:
Do we need to update `argoproj.AddToScheme` to `argoproj.Install` or is it that deprecation is only for specific kinds?

Reply:
`argoproj.AddToScheme` appears to be a totally different function, and is not deprecated.